client-app/src/common/services/url.service.ts

```diff
@@ -0,0 +1,15 @@
+import { baseUrl, locale, storeName, defaultPageSize } from "@common/constants";
```

Comment:
Why we can't define these urls in api-clients.service.ts?

Reply:
What about you? These are already defined in constants.